### PR TITLE
User guide update

### PR DESCRIPTION
**Pull request by MerlinLim**
- Restructure order of commands (shift View Command)
- Ensure e.g add commands work
- Fix grammar and Styling 
- Fix date time format

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -16,29 +16,30 @@ title: User Guide
...
 
-   * **`switch`**`dv` : Switches to deliverable mode.
+   1. **`switch`** `dv` : Switches to deliverable mode.
 
-   * **`add`**`r/stk n/Betty Clarke e/betty@email.com` : If in contact mode, adds a contact named `Betty Clarke`.
+   1. **`add`** `t/Find profile page template m/2.1.1 by/12-12-2020 12:00` : Adds a deliverable with the 
+   title `Find profile page template`, milestone `2.1.1` and deadline `12-12-2020 12:00`.
 
-   * **`delete`**`1` : If in meeting mode, deletes the 1st meeting shown.
+   1. **`delete`** `1` : Deletes the 1st deliverable shown.
```
Comment (chrystalquek) [inline docs/UserGuide.md:40]: just in case, maybe better to leave the "if in .. mode"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -51,7 +52,9 @@ Here are some example commands you can try:
 **:information_source: Notes about the command format:**<br>
 
 * Words in upper case are the parameters to be supplied by you for their respective fields.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.
+  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.<br>
+:bulb: **Tip:** If you are not sure what specific parameter to supply for any of the required fields, supply an estimate or random value as place holder.<br>
+:bulb: **Tip:** For any fields with an unspecified maximum length, the display may look untidy if you supply a long parameter.
```
Comment (chrystalquek) [inline docs/UserGuide.md:57]: more like a note.
maybe can say that the list view will be truncated for fields with long parameters

File: docs/UserGuide.md [hunk 3]
```diff
@@ -172,18 +176,32 @@ Examples:
...
+#### Viewing a deliverable: `view`
+
+Displays more details of the specified deliverable from your displayed deliverable list.
+
+Format: `view INDEX`
+* `INDEX` is the index number of the deliverable in your displayed deliverable list.
+* `INDEX` must be a positive integer.
+
+Example:
+* `view 2` views the 2nd deliverable in your displayed deliverable list.
```
Comment (chrystalquek) [inline docs/UserGuide.md:201]: actually this is a bit weird, should it be displays or some other word

File: docs/UserGuide.md [hunk 4]
```diff
@@ -255,42 +259,38 @@ Adds a meeting to your meeting list.
...
+* `CONTACTS` represents the contact(s) involved in the meeting. 
 * `CONTACTS` is a comma-separated string of names.
 * `LOCATION` is the location of the meeting.
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes:**<br>
 
-* Meeting timings may overlap as you may wish to send a representative for your clashing meetings.
+* Different meeting may have overlapping timings as you may wish to send a representative for your clashing meetings.
```
Comment (chrystalquek) [inline docs/UserGuide.md:274]: meetings

File: docs/UserGuide.md [hunk 5]
```diff
@@ -255,42 +259,38 @@ Adds a meeting to your meeting list.
...
-* `add t/Discuss app requirements from/12-12-2020 09:00 to/12-12-2020 10:00 d/Refine with business associates c/3,6,9`
+* `add t/Discuss app requirements from/12-12-2020 09:00 to/10:00 d/Refine with business associates c/Jordan Woods, Betsy Crowe`
 adds a meeting with the title `Discuss app requirements`,
-start date time `12-12-2020 09:00`, end date time `12-12-2020 10:00`, description `Refine with business associates` and contacts `3,6,9`.
-* `add t/User research review from/15-12-2020 13:00 to/15-12-2020 15:00 l/Meeting room A` 
+start date time `12-12-2020 09:00`, end time `10:00`, description `Refine with business associates` and contacts `Jordan Woods, Betsy Crowe`.
+* `add t/User research review from/15-12-2020 13:00 to/15:00 l/Meeting room A` 
 adds a meeting with the title `User research review`, start date time `15-12-2020 13:00`, 
-end date time `15-12-2020 15:00`, and location `Meeting room A`.
+end date time `15:00`, and location `Meeting room A`.
```
Comment (chrystalquek) [inline docs/UserGuide.md:289]: end time

Comment (chrystalquek) [reply docs/UserGuide.md:289]: remove comma

File: docs/UserGuide.md (hunk 3 above)
Comment (MerlinLim) [reply docs/UserGuide.md:201]: yeah it is abit weird but im not sure what else to leave it as. Its not wrong tho

File: docs/UserGuide.md (hunk 1 above)
Comment (MerlinLim) [reply docs/UserGuide.md:40]: I feel like its alright, actually if i wanted to really include I should be including that condition for "add t/deliverable..." instead

File: docs/UserGuide.md (hunk 2 above)
Comment (MerlinLim) [reply docs/UserGuide.md:57]: will ammend

File: docs/UserGuide.md (hunk 1 above)
Comment (chrystalquek) [reply docs/UserGuide.md:40]: right, ok just saw the 'sequence'

**Review by chrystalquek — APPROVED** [review]
LGTM

**Pull request merged**